http-python3.12-django5.0/Kraftfile

```diff
@@ -0,0 +1,7 @@
+spec: v0.6
+
+runtime: index.unikraft.io/official/python:3.12
```

Comment:
```diff
- runtime: index.unikraft.io/official/python:3.12
+ runtime: python:3.12
```

Reply:
Is there some explanation when to use one over the other?

Reply:
We should only use the abbreviated version in examples, i.e. this change sans the registry. It makes this example compatible with OSS unikraft. The use of the full registry is an artifact of an older version of `kraft` which did not support such abbreviation.